### ISSUE TITLE
Apply tooltips to form elements

### ISSUE THREAD

**Issue by k3davis** [neluckoff/vue3-tooltip#2]
I have a sort of niche case where I want to apply a tooltip to a form input (placeholder text won't fit). This doesn't work as-is. Additionally the inputs themselves are rendered in a `v-for` so while I found I could make the tooltip work by applying it to a wrapping `div` (for example), it complicates the code to have to do that.

I tried using the component method instead of the directive (putting the input in the component's template) and that works, but the animation is really jumpy, and the same overall complication applies -- I guess I could try applying `v-for` to the tooltip component itself but that's definitely not intuitive.

TL;DR Is it possible that this component could allow directive tooltips to work when applied to `input` tags?
